Packs/PICUS/Integrations/PICUS/PICUS.py

```diff
@@ -0,0 +1,901 @@
+import json
+import os
+import re
+import traceback
+import urllib
+
+import demistomock as demisto  # noqa: F401
+import requests
+from CommonServerPython import *  # noqa: F401
+from requests.exceptions import ConnectionError, HTTPError
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBAL VARIABLES'''
+VERIFY_SSL = not demisto.params().get('insecure', False)
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+if not demisto.params().get('proxy', False):
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+
+
+def test_module() -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client:
+
+    :type name: ``str``
+    :param name:
```

Comment:
Notice that those parameters doesn't actually being received by the function.

Reply:
Resolved.